l10n/en/brands.ftl

```diff
@@ -89,6 +89,7 @@
 ## Pocket
 
 -brand-name-pocket = Pocket
+-brand-name-read-it-later-inc = Read It Later, Inc.
```

Comment:
I wondered this so Pmac and I discussed it, but we're just going with the brands from the main l10n dir to avoid unncessary duplication - the brands are never translated, so they're more like shared constants.

Reply:
> the brands are never translated, so they're more like shared constants.

This isn't entirely true, some locales do translate certain brand terms I believe.

Reply:
Hmm... I thought that was the point of the brands file. If they can be translated then perhaps we should just have a separate one for the pocket strings. It'll be a lot shorter anyway.

Reply:
Most locales leave them as English, but there are some occurences where (iiuc) it makes more sense to translate. e.g. https://github.com/mozilla-l10n/www-l10n/blob/master/fr/brands.ftl#L65

Agree the Pocket list will be a lot smaller with less baggage :)

Reply:
- [ ] @stevejalim - add Pocket-specific brands.ftl 
- [ ] @stevejalim - see if we can remove the Mozorg/Pocket-mode check from the Fluent loader logic

Reply:
@pmac @alexgibson Done in 6ac09978c

The `-brand-name-read-it-later-inc` string is moved from `l10n/en/brands.ftl` into the new `l10n-pocket/en/brands.ftl` file and can be used as proof that the new brands file is being loaded, because the correct Read It Later, Inc string appears at the bottom of the about page.

What _is_ bugging me, though, is even though I've remapped FLUENT_LOCAL_PATH to be pocket-specific (now we don't need any mozorg Fluent files, we still need to have the IS_POCKET_MODE override in place for it to work -- @pmac what am I missing here?

Reply:
Not sure. I'll look.

Reply:
I'm wondering if it's the manipulation of `FLUENT_PATHS` in `settings/__init__.py` instead of just redefining it. I'm guessing it's not turning out as expected.